frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/hooks/useStrategyParameters.tsx

```diff
@@ -0,0 +1,94 @@
+import { type FC, useMemo } from 'react';
+import { StrategyEvaluation } from '../StrategyEvaluationChip/StrategyEvaluationChip';
+import {
+    parseParameterNumber,
+    parseParameterStrings,
+} from 'utils/parseParameter';
+import { StrategyEvaluationItem } from '../StrategyEvaluationItem/StrategyEvaluationItem';
+import type { IFeatureStrategyPayload } from 'interfaces/strategy';
+import type { CreateFeatureStrategySchema } from 'openapi';
+
+const RolloutParameter: FC<{
+    value?: string | number;
+    parameters?: (
+        | IFeatureStrategyPayload
+        | CreateFeatureStrategySchema
+    )['parameters'];
+    hasConstraints?: boolean;
+    displayGroupId?: boolean;
+}> = ({ value, parameters, hasConstraints, displayGroupId }) => {
+    const percentage = parseParameterNumber(value);
+
+    const explainStickiness =
+        typeof parameters?.stickiness === 'string' &&
+        parameters?.stickiness !== 'default';
+    const stickiness = explainStickiness ? (
+        <>
+            with <strong>{parameters.stickiness}</strong>
+        </>
+    ) : (
+        ''
+    );
+
+    return (
+        <StrategyEvaluationItem type='Rollout %'>
+            <StrategyEvaluation label={`${percentage}%`} /> of your base{' '}
+            {stickiness}
+            <span>
+                {hasConstraints ? 'who match constraints ' : ' '}
+                is included.
+            </span>
+            {/* TODO: displayGroupId */}
+        </StrategyEvaluationItem>
+    );
+};
+
+export const useStrategyParameters = (
+    strategy: IFeatureStrategyPayload | CreateFeatureStrategySchema,
+    displayGroupId?: boolean,
+) => {
+    const { constraints } = strategy;
+    const { parameters } = strategy;
+    const hasConstraints = Boolean(constraints?.length);
+    const parameterKeys = parameters ? Object.keys(parameters) : [];
+    const mapPredefinedStrategies = (key: string) => {
+        if (key === 'rollout' || key === 'Rollout') {
```

Comment:
How about just upper/lowercasing here to avoid failing on other weird casings? 
```suggestion
        if (key.toLowerCase() === 'rollout') {
```